### ISSUE TITLE
WCAG2ICT v20251211 updates

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/matf#285]
We need to go through the December 2025 publication of WCAG2ICT and check for changes that we need to align with.

https://www.w3.org/TR/2025/NOTE-wcag2ict-22-20251211/

https://services.w3.org/htmldiff?doc1=https://www.w3.org/TR/2025/NOTE-wcag2ict-22-20251211/&doc2=https://www.w3.org/TR/2025/NOTE-wcag2ict-22-20250821/

**Comment by JJdeGroot** [comment]
Discussed in MATF meeting on 3 December 2025.

<details>
<summary>Minutes for 3 December 2025</summary>

[Source](https://www.w3.org/2025/12/03-matf-minutes.html#7f9b)

~~~text
JJ: Summarises the status of the v4.1.0 EN standard and WCAG2ICT. As a group, we probably need to check whether our guidance should be aligned.
~~~
</details>

**Comment by TanyaVanWorkum** [comment]
Link to the blog with changes in EN 301 549 v4.1.0 outlined: https://abra.ai/blog/mobile-app-accessibility-en-301-549-v4-1-0

**Comment by detlevhfischer** [comment]
Hi Tanya, that is a good summary of the changes, very useful, many thanks to Abra!

I have one disagreement regarding the way you present 11.1.4.4 Resize Text. The EN in draft version 4.0 clearly states:

> This means that the non-web software provides some means for enlarging the text 200 percent (**zoom or otherwise)**
without loss of content or functionality, **or that the non-web software works with the platform features to
satisfy this success criterion** (_my emphasis_).  

I would count the OS Zoom functions in iOS and Android as platform features. The way you present 11.1.4.4 appears to suggest that the only way to meet it is by supporting a change of text size in the OS settings. This is certainly recommended but in my reading, relying on the OS Zoom function is an acceptable way of meeting 11.1.4.4.

Of course, this hinges also on the way the WCAG definition of assistive technology is interpreted. There, [assistive technology](https://www.w3.org/TR/WCAG22/#dfn-assistive-technologies) is defined as 
> hardware and/or software that acts as a user agent, or along with a mainstream user agent

...and the examples point to stand-alone software like screen magnifiers, screen readers and voicve inout software rather than to OS accessibility features like the built-in OS zoom function. [Note 3](https://www.w3.org/TR/WCAG22/#h-note-67) points out that 
> The distinction between mainstream user agents and assistive technologies is not absolute. Many mainstream user agents provide some features to assist individuals with disabilities. 

...which to me implies that "some features of mainstream user agents" (again, lioke the zoom function) do not count as assisitive technology in the sense of the WCAG definition.

**Comment by TanyaVanWorkum** [comment]
Thanks for your feedback, @detlevhfischer . For clarity, I replaced “This can be achieved either through in-app mechanisms or by working correctly **with system text size settings**” with “This can be achieved either through in-app mechanisms or by working correctly **with platform features (for example, system text size settings).**” I also added the following clarification:

> It is also important to consider Clause 11.7 User preferences. In a mobile context, this means that when a documented platform accessibility feature, such as text resizing, is provided by the operating system and enabled by the user, apps are expected to respect that setting and scale text accordingly. Only apps that are deliberately designed to be isolated from the platform are exempt from this requirement.

Clause 11.7 is not new in v4.1.0, but its wording and notes make the intent clearer when interpreting requirements such as 11.1.4.4 Resize Text. While relying on OS Zoom function may satisfy the resize requirement itself under 11.1.4.4, Clause 11.7 places an additional obligation on non-web software to follow documented platform accessibility preferences where these exist.

In particular, NOTE 4 of Clause 11.7 states:

> Features that might be documented as accessibility features for a platform include colour filters, contrast, text size, pointer size, and text cursor.

Based on this, our interpretation is that relying solely on OS Zoom may be sufficient for meeting 11.1.4.4, but may not be sufficient to meet Clause 11.7, where text size is explicitly treated as a documented platform accessibility feature. For this reason, in our audits we always verify whether text scales correctly when the system font size is increased to 200% (or as close as the platform allows). 

This interpretation applies equally to other documented platform accessibility features mentioned in Clause 11.7, not only text size.

**Comment by detlevhfischer** [comment]
@TanyaVanWorkum Completely agree - if body text does not scale when larger text size is selected in the OS we would also fail this is in 11.7 (but generally pass 11.1.4.4). The difficult questions (and grey areas) remain both regarding the scope of text-sizing (it has been pointed out that some text cannot (or should) not resize because it would truncate or mess up the interface), and regarding whether consequences of user customization should be taken into account at all (in the web space, a typical case woul be controls or control states like checkbox / checkbox marks disappearing when setting custom colors - would you then fail the future 9.7?). 
And the general question with loose lists like "features include 1, 2, 3, 4, ..." is whether 11.7 can be deemed already satisfied if just one of these features can be set; some but not all can be set; or all that the OS supports must be settable by an app. So there seems to be a lot of grey area still.
